packages/next-auth/src/providers/appdirect.js

```diff
@@ -0,0 +1,27 @@
+/** @type {import(".").OAuthProvider} */

```

Comment:
It would be awesome if you could provide this file as a `.ts` file
We're trying to convert all the providers from `.js` to `.ts`, so naturally, all new providers should be in `.ts` 😄